

export async function handleFetchStudies(client) {
	try {

	} catch (error) {
		return {
			errors: error,
			success: false,
			data: null,
			msg: "Internal server error",
		}
	}
}
